core/app/[locale]/(default)/account/(tabs)/orders/page-data.ts

```diff
@@ -241,20 +247,37 @@ const addProductAttributesToShippingConsignments = async (consignments: Shipping
   return shippingConsignments;
 };
 
+type OrdersFiltersInput = VariablesOf<typeof CustomerAllOrders>['filters'];
+type OrderStatus = NonNullable<OrdersFiltersInput>['status'];
+type OrderDateRange = NonNullable<OrdersFiltersInput>['dateRange'];
+
 interface CustomerOrdersArgs {
   after?: string;
   before?: string;
   limit?: number;
+  filterByStatus?: OrderStatus;
+  filterByDateRange?: OrderDateRange;
 }
 
 export const getCustomerOrders = cache(
-  async ({ before = '', after = '', limit = 2 }: CustomerOrdersArgs) => {
+  async ({
+    before = '',
+    after = '',
+    filterByStatus,
+    filterByDateRange,
+    limit = 2,
+  }: CustomerOrdersArgs) => {
```

Comment:
🍹 We might be able to get away with using the variables as the arguments. No action, but just a thought.
```suggestion
  }: VariablesOf<typeof CustomerAllOrders>) => {
```

Reply:
Yeah, I agree. But for now I'd prefer to leave it and update later since we use quite similar number of arguments across all functions where we have pagination data. It might be worth to go and replace all of them at once🤔